plugins/telemetry/telemetry_impl.cpp

```diff
@@ -489,6 +509,39 @@ void TelemetryImpl::process_heartbeat(const mavlink_message_t &message)
     }
 }
 
+void TelemetryImpl::process_statustext(const mavlink_message_t &message)
+{
+    mavlink_statustext_t statustext;
+    mavlink_msg_statustext_decode(&message, &statustext);
+
+    Telemetry::StatusText::StatusType type;
+
+    switch (statustext.severity) {
+        case MAV_SEVERITY_WARNING:
+            type = Telemetry::StatusText::StatusType::WARNING;
+            break;
+        case MAV_SEVERITY_CRITICAL:
+            type = Telemetry::StatusText::StatusType::CRITICAL;
+            break;
+        default:
+            type = Telemetry::StatusText::StatusType::INFO;
```

Comment:
I guess that's a good default for now to just use `INFO` if we don't know it. However, I could catch `INFO` properly in a `case` and then for default emit a `LogWarn() << "Unknown StatusText severity"` so that we are warned if we ever get a severity that we we don't expect.

Reply:
Cool. I thought about leaving a UNKNOWN status type as a default, but then I'd have to change the proto file. For now I'm doing something like this:
```
    switch (statustext.severity) {
        case MAV_SEVERITY_WARNING:
            type = Telemetry::StatusText::StatusType::WARNING;
            break;
        case MAV_SEVERITY_CRITICAL:
            type = Telemetry::StatusText::StatusType::CRITICAL;
            break;
        case MAV_SEVERITY_INFO:
            type = Telemetry::StatusText::StatusType::INFO;
            break;
        default:
            LogWarn() << "Unknown StatusText severity";
            type = Telemetry::StatusText::StatusType::INFO;
            break;
    }
```
I still set type to info if we get an unknown value, since we'd like to see it in our app. 
Let me know if I should leave it like this, or actually set the type as UNKNOWN.

Reply:
Yes exactly, that's what I meant.